src/api/customer/customer-type.ts

```diff
@@ -45,4 +45,73 @@ export type CustomerCreationRequestHandler = RequestHandler<
         customerCreated: CustomerAttributes,
         customerToRespond: CustomerResponse
     }
+>
+
+// ---------------------------------------------------------------------------------------------------------------------- //
+
+type CustomerError = {
+    statusCode: number;
+    error: string;
+    message: string;
+    validation: {
+        params: {
+            source: string;
+            keys: Array<string>;
```

Comment:
Só por curiosidade, se quiser ser menos verboso poderia fazer assim:

```suggestion
            keys: string[];
```